kubetest2-kind/deployer/deployer.go

```diff
@@ -90,8 +101,8 @@ func bindFlags(d *deployer) *pflag.FlagSet {
 	return flags
 }
 
-// assert that deployer implements types.Deployer
-var _ types.Deployer = &deployer{}
+// assert that deployer implements types.DeployerWithKubeconfig
```

Comment:
Is there any deployer that doesn't have kubeconfig?

Reply:
Don't think so

Reply:
I think we should make this a standard part of the API for deployer, a requirement for kubetest2, but that can be a follow-up.

Reply:
ack